src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/security/SecurityConfiguration.kt

```diff
@@ -37,7 +37,7 @@ private val logger = loggerFor<SecurityConfiguration<*>>()
 @EnableWebSecurity
 @ConditionalOnWebApplication
 @EnableGlobalMethodSecurity(prePostEnabled = true)
-class SecurityConfiguration<S : Session> : WebSecurityConfigurerAdapter() {
```

Comment:
`WebSecurityConfigurerAdapter` is deprecated after moving to a later version of Spring.